package.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
-    "@escape.tech/mookme": "^2.0.0-beta.1"
+    "@escape.tech/mookme": "^2.0.0-beta.1",
+    "react-dom": "^18.1.0"
```

Comment:
This probably got here by mistake, you can remove it with `npm uninstall react-dom` in the root directory of the repo

Reply:
you need to remove it from the root